### ISSUE TITLE
Whipper fails to analyze drive

### ISSUE THREAD

**Issue by copyread** [whipper-team/whipper#447]
First time trying whipper. After entering the command `whipper drive analyze`, ~2 minutes of seemingly normal drive operation follow, then it makes a weird, stuttering repetitive noise for 10 seconds, then throws this error:
```
CRITICAL:whipper.command.main:exception TypeError at /usr/lib/python3.8/site-packages/whipper/program/cdparanoia.py:606: failed(): sequence item 0: expected str instance, bytes found
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/whipper/common/task.py", line 89, in _read
    self._done()
  File "/usr/lib/python3.8/site-packages/whipper/common/task.py", line 107, in _done
    self.failed()
  File "/usr/lib/python3.8/site-packages/whipper/program/cdparanoia.py", line 606, in failed
    output = "".join(self._output)
TypeError: sequence item 0: expected str instance, bytes found

```
OS: Manjaro 18.1.5
Drive: HL-DT-STDVDRAM GP57EB40
Whipper version: 0.9.0-1, installed from the community repository

**Comment by copyread** [comment]
also, CD that I used: Deep Purple - Slaves and Masters. Can be used to configure AccurateRip for EAC.

**Comment by copyread** [comment]
Forgot about the log.
[whipper.log.gz](https://github.com/whipper-team/whipper/files/4047653/whipper.log.gz)

**Comment by copyread** [comment]
It seems installing whipper broke something in my system, I can't even rip a DVD I successfully ripped before with `dvdbackup`, now getting read errors.

I didn't break my drive, I still can get a bit-perfect copy of Slaves and Masters (according to AccurateRip) on EAC @ Windows.

**Comment by JoeLametta** [comment]
Hi, this appears to be a duplicate of issue #431 which was fixed in pull request #434 (commit 29ee670b7fa6ec7402c774091b893d5414e59bc0). We still need to tag an updated version of whipper...

> It seems installing whipper broke something in my system, I can't even rip a DVD I successfully ripped before with `dvdbackup`, now getting read errors.

Sorry but I think this isn't related to / caused by this issue.
